.github/workflows/lighthouserc.json

```diff
@@ -3,7 +3,8 @@
     "collect": {
       "staticDistDir": "./_site",
       "autodiscoverUrlBlocklist": [
-        "conduct.html"
+        "conduct.html",
+        "https://docs.jupyter.org/en/latest/"
```

Comment:
I think it is entering it from some redirect, maybe from try `"documentation.html"` or "documentation"` instead?